client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -377,19 +384,60 @@ class CheckoutThankYouHeader extends PureComponent {
 					<div className="checkout-thank-you__header-copy">
 						<h1 className="checkout-thank-you__header-heading">{ this.getHeading() }</h1>
 
-						<h2 className="checkout-thank-you__header-text">{ this.getText() }</h2>
+						{ primaryPurchase && isSimplified ? (
```

Comment:
> Otherwise a domain purchase + isSimplified would show the "Your site is now on the..." text

While we don't explicitly check for a plan here, `isSimplified` is based on the `upgradeIntent` query arg - and you can only get one via business/premium plan upgrade nudges.

Reply:
Still that makes sense so I'm updating it